larq/metrics.py

```diff
@@ -62,7 +63,54 @@ def get_training_metrics():
     return _GLOBAL_TRAINING_METRICS
 
 
-class FlipRatio(Metric):
+class _CompatibilityMetric(Metric):
+    r"""Metric with support for both 1.13 and 1.14+
+    """
+
+    def get_config(self):
```

Comment:
I don't think we need to overwrite this since this is the default anyway:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/metrics.py#L199-L201